sdk/core/core-amqp/src/requestResponseLink.ts

```diff
@@ -84,6 +85,12 @@ export class RequestResponseLink implements ReqResLink {
 
     const aborter: AbortSignalLike | undefined = options.abortSignal;
 
+    // Adding a backup message_id in case it is `undefined`,
+    // helps in differentiating the responses inside `messageCallback` defined a few lines below.
```

Comment:
Consider having comments that speak to the concept behind the change instead of explicit changes that are being made which can be inferred by looking at the code.

For example, "Adding a backup message_id in case it is `undefined`" can be inferred by looking at the if block.
"inside `messageCallback` defined a few lines below" does not help much as one can do a text search to see where it is getting used


```suggestion
    // If message_id is not already set on the request, set it to a unique value 
    // This helps in determining the right response for current request among multiple incoming messages
```

Reply:
Thank you!!